### PR TITLE
docs(benchpress): expand benchmark running instructions

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

**Review by mhevery — APPROVED** [review]

File: modules/benchmarks/README.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
 
-# e.g. "ng2" tree benchmark:
+# e.g. tree benchmark implemented with Angular (view engine in dev mode):
 yarn bazel run modules/benchmarks/src/tree/ng2:devserver
+
+# e.g. tree benchmark implemented with Angular (ivy in dev mode):
+yarn bazel run modules/benchmarks/src/tree/ng2:devserver --define=compile=aot
+```
+
+## Run in the browser (prod mode)
```
Comment (JoostK) [inline modules/benchmarks/README.md]: There doesn't appear to be a `prodserver` target in any of the benchmarks:

```
ERROR: no such target '//modules/benchmarks/src/tree/ng2:prodserver': target 'prodserver' not declared in package 'modules/benchmarks/src/tree/ng2'
```

File: modules/benchmarks/README.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
 ```bash
-yarn bazel run modules/benchmarks/src/tree/{name}:devserver
+yarn bazel run modules/benchmarks/src/{benchmark_name}/{impl_name}:devserver
 
-# e.g. "ng2" tree benchmark:
+# e.g. tree benchmark implemented with Angular (view engine in dev mode):
 yarn bazel run modules/benchmarks/src/tree/ng2:devserver
+
+# e.g. tree benchmark implemented with Angular (ivy in dev mode):
+yarn bazel run modules/benchmarks/src/tree/ng2:devserver --define=compile=aot
```
Comment (IgorMinar) [inline modules/benchmarks/README.md]:
```suggestion
yarn bazel run modules/benchmarks/src/tree/ng2:devserver --config=ivy
```

File: modules/benchmarks/README.md [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+## Run in the browser (prod mode)
+
+```bash
+yarn bazel run modules/benchmarks/src/{benchmark_name}/{name}:prodserver
+
+# e.g. tree benchmark implemented with Angular (view engine in prod mode):
+yarn bazel run modules/benchmarks/src/tree/ng2:prodserver
+
+# e.g. tree benchmark implemented with Angular (ivy in prod mode):
+yarn bazel run modules/benchmarks/src/tree/ng2:prodserver --define=compile=aot
```
Comment (IgorMinar) [inline modules/benchmarks/README.md]:
```suggestion
yarn bazel run modules/benchmarks/src/tree/ng2:prodserver --config=ivy
```

File: modules/benchmarks/README.md [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
+## Run benchpress
+
+```bash
+yarn bazel run modules/benchmarks/src/{benchmark_name}/{name}:perf
+
+# e.g. tree benchmark implemented with Angular (view engine):
+yarn bazel run modules/benchmarks/src/tree/ng2:perf
+
+# e.g. tree benchmark implemented with Angular (ivy):
+yarn bazel run modules/benchmarks/src/tree/ng2:perf --define=compile=aot
```
Comment (IgorMinar) [inline modules/benchmarks/README.md]:
```suggestion
yarn bazel run modules/benchmarks/src/tree/ng2:perf --config=ivy
```

File: modules/benchmarks/README.md [hunk 5]
```diff
@@ -1,12 +1,39 @@
 # How to run the benchmarks locally
 
-## Run in the browser
+## Run in the browser (dev mode)
```
Comment (IgorMinar) [inline modules/benchmarks/README.md]: I don't believe that we have a dev mode for running benchmarks. The ts_devserver rule is used to provide devserver which hosts an optimized bundle used for profiling.

Also as Joost noted below there is no prodserver so the whole section you added shouldn't have been added.

**Review by IgorMinar — CHANGES_REQUESTED** [review]

**Comment by petebacondarwin** [comment]
@pkozlowski-opensource - hope you are fine and dandy! Shall we just close this PR, since it hasn't been touched for almost a year, or can it be resurrected?

**Comment by AndrewKushnir** [comment]
@pkozlowski-opensource hope you're doing great!
FYI, I'm closing this PR for now, feel free to reopen once you get a chance to look at it again. Thank you.

**Pull request closed without merging**